modules/javafx.controls/src/main/java/javafx/scene/control/SelectionModel.java

```diff
@@ -199,8 +199,8 @@ public SelectionModel() { }
 
     /**
      * <p>Convenience method to inform if the given index is currently selected
-     * in this SelectionModel. Is functionally equivalent to calling
-     * <code>getSelectedIndices().contains(index)</code>.
+     * in this SelectionModel.  It will return true when at least one cell
+     * is selected in the specified row index.
      *
```

Comment:
Well, I would go a step (actually 2 steps :) further:

A:

- the doc error was that a method in the base class specifies its behavior with api from a sub: can't and should be done because it doesn't know its subs plus there are subs that don't have such api (like SingleSelection model). 
- the fix is to replace the description with a similar error: there is no notion of "cells in a row" in the base and also not in many of its subs

I would suggest to simply remove the old sentence without replacing it with anything: it has the notion of multiple selections without actually api to access those multiples, selected could stand for its own and might be fleshed out by subclasses.

B: 

we need a CSR anyway, so might cleanup a bit further: this method is _not_ a convenience method, quite on the contrary - for clients coding against this base class, it's the _only_ means to get hold of the selected indices (by looping through all indices in the valid range and calling it)